### PR TITLE
refactored reference browsertests to T87759

### PR DESCRIPTION
**Pull request by WMDE-Fisch**
https://phabricator.wikimedia.org/T87759
https://gerrit.wikimedia.org/r/#/c/190799/

### CODE REVIEW

File: tests/browser/features/step_definitions/reference_steps.rb [hunk 1]
```diff
@@ -57,72 +36,29 @@
...
+  expect(on(ItemPage).add_reference_element.when_visible).to be_visible
 end
 
 Then(/^Reference add button should not be there$/) do
-  on(ItemPage).add_reference_element.when_not_visible
-end
-
-Then(/^Reference add button should be hidden$/) do
-  on(ItemPage).add_reference_element.when_not_visible
+  expect(on(ItemPage).add_reference_element.when_not_visible).to_not be_visible
```
Comment (zeljkofilipin) [inline tests/browser/features/step_definitions/reference_steps.rb]: Just a note, `to_not` can be replaced with `not_to`. I think it makes the code more readable, but it is optional.

**Comment by tobijat** [comment]
This should only be merged together with https://gerrit.wikimedia.org/r/#/c/190799/

**Comment by thiemowmde** [comment]
Needs rebase.

**Comment by thiemowmde** [comment]
+2! I tested this along with https://gerrit.wikimedia.org/r/252673 on my local machine and all reference.feature tests pass.

**Comment by WMDE-Fisch** [comment]
finally :-)

**Pull request merged**